.mlc_config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "aliveStatusCodes": [
+    403,
+    200,
+    206
+  ],
+  "httpHeaders": [
+    {
+      "urls": [
+        "https://github.com/",
+        "https://guides.github.com/",
+        "https://help.github.com/",
+        "https://docs.github.com/"
+      ],
+      "headers": {
+        "Accept-Encoding": "zstd, br, gzip, deflate"
+      }
+    }
```

Comment:
Whenever checking links, to these URLs, send these header options.
This is a config that I've found in the markdown-link-checker repo
and works great so far in other link checking deployments that I have.